commit 9be4ec7ebf2d759114a98b0f532171f0e9e5768d
Author: fxcoudert <fxcoudert@138bc75d-0d04-0410-961f-82ee72b054a4>
Date:   Sun Sep 14 08:05:43 2014 +0000

    	PR target/61407

    	* config/darwin-driver.c (darwin_find_version_from_kernel): Removed
    	kernel version check to avoid incrementing it after every major OS X
    	release.
---
Extracted from upstream r215251.

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=61407
https://gcc.gnu.org/viewcvs/gcc?view=rev&rev=215251

 gcc/config/darwin-driver.c | 2 --
 1 file changed, 2 deletions(-)

diff --git gcc/config/darwin-driver.c gcc/config/darwin-driver.c
index 48eaf19..5b3991c 100644
--- gcc/config/darwin-driver.c
+++ gcc/config/darwin-driver.c
@@ -57,8 +57,6 @@ darwin_find_version_from_kernel (char *new_flag)
   version_p = osversion + 1;
   if (ISDIGIT (*version_p))
     major_vers = major_vers * 10 + (*version_p++ - '0');
-  if (major_vers > 4 + 9)
-    goto parse_failed;
   if (*version_p++ != '.')
     goto parse_failed;
   version_pend = strchr(version_p, '.');
